runtime/src/launcher/js/launcher.js

```diff
@@ -216,9 +216,15 @@ konan.moduleEntry = function (args) {
             throw new Error('Could not find the wasm attribute pointing to the WebAssembly binary.');
         }
         const filename = document.currentScript.getAttribute("wasm");
-        fetch(filename)
-            .then(response => response.arrayBuffer())
-            .then(arraybuffer => instantiateAndRun(arraybuffer, [filename]));
+        if (typeof WebAssembly.instantiateStreaming === 'function') {
+            linkJavaScriptLibraries();
+            WebAssembly.instantiateStreaming(fetch(filename), konan_dependencies)
+              .then(resultObject => invokeModule(resultObject.instance, [filename]));
```

Comment:
May be extract this scenarion as a third function, similar to instantiateAndRun and instantiateAndRunSync.

Reply:
Should it take only filename as an argument? Like
```js
// Instantiate module in Browser as a promise of streaming instantiation.
function instantiateAndRunStreaming(filename) {
    linkJavaScriptLibraries();
    WebAssembly.instantiateStreaming(fetch(filename), konan_dependencies)
      .then(resultObject => invokeModule(resultObject.instance, [filename]));
}
```

Reply:
that's okay, I suppose.